tests/test_helpers/test_serializers.py

```diff
@@ -35,7 +37,13 @@ def test_encode_decode_i():
         "list_of_ints": [1, 2, 3],
         "list_of_str": ["1234", "234234"],
         "list_of_floats": [1.1, 2.2, 3.0],
-        "nested_dict": {"a": b"test", "b": 10, "c": [b"1", b"2"]},
+        "nested_dict": {
+            "a": b"test",
+            "b": 10,
+            "c": [b"1", b"2"],
```

Comment:
Please add list of ints, floats, bools, strings as well.

Reply:
updated